test/rules/unnecessary_string_escapes_test.dart

```diff
@@ -0,0 +1,210 @@
+// Copyright (c) 2022, the Dart project authors. Please see the AUTHORS file
```

Comment:
I think this should be 2023?  (Or 2020 if you've been keeping the copyright from the migrated tests?)

Reply:
done!